raml/sfdc_api.raml

```diff
@@ -0,0 +1,52 @@
+# RAML 0.0
+title:
+version: v1
+baseUrl: http://example.com/api
+/alert:
+  post:
+    description: Create or update Case in SFDC
+    responses:
+      200:
+    body:
+      application/json:
+        schema |
+          {"$schema": “http://json-schema.org/schema”,
+            "type":"object"
+            "description": "Alert Data"
+            "properties": {
+              "description":   {"type": "json"},
+              "alertId":       {"type": "string"},
+              "subject":       {"type": "string"},
+              "environment":   {"type": "string"},
+              "alertPriority": {"type": "string"},
+              "alertHost":     {"type": "string"},
+              "alertService":  {"type": "string"}
+          example: |
+            alert: {
+            {
+              "description":   {
+                "affected_hosts": [
+                  "host=ic3-ctl01-scc1", 
+                  "host=ic3-ctl01-scc2", 
+                  "host=ic3-ctl01-scc"
+                ], 
+                "description": " Test \n
+                    The CPU usage is too high (controller node). \n
+                    (CRITICAL, rule=avg(cpu_idle)=5, current=1.65, host=ic3-ctl01-scc) \n
+                    No datapoint have been received over the last 60 seconds \n
+                    (UNKNOWN, rule=min(fs_space_percent_free[fs=/var/log])2, \n
+                    current=-1.00, host=ic3-ctl01-scc1), \n
+                "long_date_time": "Fri Dec 2 14:14:53 UTC 2016",
+                "service": "glance1"
+              },
+              "alertId":       "a2X4C000000Cf0V--glance1--00-global-clusters-SIT12",
```

Comment:
alertId is SFDC artifact and it is used to understand do we need to create new alert or update existing

E.g. if service is flapping and send up/down alerts many times with same alert ID the only one alert object will be created in SFDC, all other will be attached as feed_items

It was  done in this to avoid flooding.